tailcfg/tailcfg.go

```diff
@@ -1385,6 +1385,12 @@ type MapRequest struct {
 	//     * "warn-router-unhealthy": client's Router implementation is
 	//       having problems.
 	DebugFlags []string `json:",omitempty"`
+
+	// ConnectionHandle, if non-empty, is an opaque string sent by the client that
```

Comment:
Do we want a "ForTest" suffix on this name? Will it have any non-test uses in the future?

Reply:
Sure. It could possibly be useful outside of tests but no specific use case comes to mind so I'll suffix it.